src/sonic-device-data/tests/permitted_list

```diff
@@ -168,7 +168,6 @@ ifp_inports_support_enable
 port_flex_enable
 pdma_descriptor_prefetch_enable
 pktdma_poll_mode_channel_bitmap
-warmboot_knet_shutdown_mode
```

Comment:
w6010 is a centec platform, why do we change brcm config in this pr?

Reply:
This revision is reverted. A background is that our customer is compiling 6010 based on our PR link in their compile environment. When they are compiling, if warmboot_knet_shutdown_mode exist in the permitted_list, the compiling will stop. This was a temporary solution for the customer.